content/docs/partnerships.mdx

```diff
@@ -154,15 +154,17 @@ HashiCorp Terraform has an extremely wide community of users and contributors an
 
 Vendors who choose not to support their provider and prefer to make it a community-supported provider will no longer be listed as Verified.
 
+-> **Network Infrastructure Automation (NIA):**  These integrations leverage Consul’s service catalog to seamlessly integrate with Consul-Terraform-Sync (CTS) to automate changes in network infrastructure via a publisher-subscriber method. More details can be found [here](https://www.consul.io/docs/integrate/nia-integration).
```

Comment:
Nit: I think its preferable to avoid "here" links for accessibility reasons.

```suggestion
-> **Network Infrastructure Automation (NIA):**  These integrations leverage Consul’s service catalog to seamlessly integrate with Consul-Terraform-Sync (CTS) to automate changes in network infrastructure via a publisher-subscriber method. More details can be found in the [NIA Integration Program](https://www.consul.io/docs/integrate/nia-integration) documentation.
```